packages/@aws-cdk/aws-codedeploy/lib/lambda/deployment-config.ts

```diff
@@ -1,23 +1,7 @@
 import { Construct } from 'constructs';
-import { BaseDeploymentConfig, BaseDeploymentConfigOptions, ComputePlatform, IBaseDeploymentConfig } from '../base-deployment-config';
+import { DeploymentConfig, BaseDeploymentConfigOptions, ComputePlatform, IDeploymentConfig } from '../base-deployment-config';
 import { TrafficRouting } from '../traffic-routing-config';
 
-/**
- * The Deployment Configuration of a Lambda Deployment Group.
- *
- * If you're managing the Deployment Configuration alongside the rest of your CDK resources,
- * use the {@link LambdaDeploymentConfig} class.
- *
- * If you want to reference an already existing deployment configuration,
- * or one defined in a different CDK Stack,
- * use the {@link LambdaDeploymentConfig#fromLambdaDeploymentConfigName} method.
- *
- * The default, pre-defined Configurations are available as constants on the {@link LambdaDeploymentConfig} class
- * (`LambdaDeploymentConfig.AllAtOnce`, `LambdaDeploymentConfig.Canary10Percent30Minutes`, etc.).
- */
-export interface ILambdaDeploymentConfig extends IBaseDeploymentConfig {
-}
```

Comment:
Same as above with ECS, this interface already existed in older versions:
https://github.com/aws/aws-cdk/blob/v2.44.0/packages/%40aws-cdk/aws-codedeploy/lib/lambda/deployment-config.ts#L13-L16